cloud/deployment/fromfile/fromfile.go

```diff
@@ -657,3 +681,18 @@ func checkEnvVars(deploymentFromFile *inspect.FormattedDeployment, action string
 func isValidExecutor(executor string) bool {
 	return executor == deployment.CeleryExecutor || executor == deployment.KubeExecutor
 }
+
+// temporary code
+func transformDeploymentType(deploymentType string) astroplatformcore.DeploymentType {
+	var transformedDeploymentType astroplatformcore.DeploymentType
+	if deploymentType == "STANDARD" || deploymentType == "standard" {
+		transformedDeploymentType = astroplatformcore.DeploymentTypeSTANDARD
+	}
+	if deploymentType == "DEDICATED" || deploymentType == "dedicated" {
+		transformedDeploymentType = astroplatformcore.DeploymentTypeDEDICATED
+	}
+	if deploymentType == "HYBRID" || deploymentType == "hybrid" {
```

Comment:
nit to end execution faster
```suggestion
	else if deploymentType == "DEDICATED" || deploymentType == "dedicated" {
		transformedDeploymentType = astroplatformcore.DeploymentTypeDEDICATED
	}
	else if deploymentType == "HYBRID" || deploymentType == "hybrid" {
```

Reply:
fixed with a switch statement